src/routes/products/[id([0-9]+)]/edit.svelte

```diff
@@ -190,14 +190,14 @@
   }
 
   async function saveChanges() {
-    if (!product.name) {
+    if (!product.name || /^\s*$/.test(product.name)) {
```

Comment:
```suggestion
    if (!product.name.trim()) {
```

Reply:
Also, if `product.name` can be `null`, the RegEx test can deal with it. No need to check it first

Reply:
But it will say `false`, while I want it to say `true` in that "true, this is a bad name"